### DISCUSSION TITLE
SaltStack Issue - High Vulnerability involving "salt-master-3003-1.el7"

### DISCUSSION THREAD

**Discussion by Mr2Ez** [Security-Onion-Solutions/securityonion#6807] [Unsupported Versions] [answered]
Hope all is well!

I'm scanning Security Onion 2 instances within my environment w/ Nessus Professional (all on the latest version and updated using "sudo soup") and I'm receiving a SaltStack vulnerability...please see attached screenshot.

Is there any plan on updating this package or should I ask for an exception from my organization long-term? Tried updating a test VM manually to test functionality with the new version, but there is a version lock when I try to do so. Any info would be much appreciated....my Cyber folks are giving me some trouble since it's a high vulnerability.

![ss_saltmaster_vuln](https://user-images.githubusercontent.com/20463554/148718836-b59dbdb7-e9c1-4fc2-8f97-f6e7cc792e6c.PNG)

-------------------------------------------------------------------------------------------------------------------------------------------------

Also, I wanted ask if there is any plan to upgrade the following to "2.17.1" versus "2.17.0" (These are flagging as medium vulnerabilities due to "log4j"):

/var/lib/docker/overlay2/d240581c15796739f5523d37f8986a1af7146191c45d02f1943c29b8e906ab91/diff/usr/share/logstash/logstash-core/lib/jars/log4j-core-2.17.0.jar

/var/lib/docker/overlay2/d4579d12cd24ddbf875676c9590f294632e2f0048515dc2ef28201b538da58f2/diff/usr/share/logstash/logstash-core/lib/jars/log4j-core-2.17.0.jar

/var/lib/docker/overlay2/e99a5d345464d6527f90fb70feff8cf7bc262166b22422f9a810c72c31257de8/merged/usr/share/elasticsearch/bin/elasticsearch-sql-cli-7.16.2.jar

/var/lib/docker/overlay2/7e7a8585e02dbc326bcd90efc081449a1c9d4151254121972ce9441d7fd95c82/diff/usr/share/elasticsearch/bin/elasticsearch-sql-cli-7.16.2.jar

/var/lib/docker/overlay2/0cc1b4acf448fd22c037e8f31b296be32426387592aacc23ebe5ffb7aebe47af/merged/usr/share/logstash/logstash-core/lib/jars/log4j-core-2.17.0.jar

/var/lib/docker/overlay2/5746c92e852a259b7122bd7bb6ffbe11dcddc95c991cb27257ccda4394061ff7/merged/usr/share/elasticsearch/bin/elasticsearch-sql-cli-7.16.2.jar

/var/lib/docker/overlay2/7b7b62a9c982ce299e7695063c423638321566a273e854ecb845da404fdeac75/diff/usr/share/elasticsearch/bin/elasticsearch-sql-cli-7.16.2.jar

Thanks in advance for any info!

-Jordan

**Comment by TOoSmOotH** [comment, marked as answer]
The salt vulnerability does not impact us but we will go ahead and update it anyway for .100. We don't run on windows nor do we use external sources for files so there is not exposure at this time.

The older 7.16.0 related findings are old docker containers. We leave the previous docker containers there in case there needs to be a roll back. You can use the command "docker system prune -a -f" if you want to remove them. The .100 release should also include 7.16.3 of the elastic components. Elastic releases that on Jan 13th so .100 should be at some point after that. Once you upgrade to .100 I am sure you will see the same old warning for 7.16.2 jar files and you will need to run the above again.

**Reply by Mr2Ez** [reply]
Thank you for your reponse, it is much appreciated! I will wait for .100 and also run "docker system prune -a -f" to remove the old docker containers that are flagging on Nessus. Have a great one.
